### ISSUE TITLE
Left-over lock files?

### ISSUE THREAD

**Issue by hadess** [flatpak/flatpak#1263]
```
$ find ~/.local/share/flatpak/system-cache/ -name "repo-*-lock" | wc -l
1545
```

All the files are named something like "repo-9XQGTY-lock".

**Comment by hadess** [comment]
All those lock files are 0 bytes.

**Comment by hadess** [comment]
There were still as many lock files today as there were when I filed this bug, so the problem is most likely solved, closing.

**Comment by mwleeds** [comment]
Reopening because I have lots of these files with recent timestamps (using Flatpak 1.6.2).

**Comment by hadess** [comment]
```
$ find ~/.local/share/flatpak/system-cache/ -name "repo-*-lock" | wc -l
0
```

**Comment by mwleeds** [comment]
@hadess do you have a collection ID configured on any remote? The code paths are pretty different depending on that.

Anyway, with pop-shop (io.elementary.appcenter) linked against Flatpak 1.6.x it created these lock files on startup every time and now with it linked against Flatpak 1.8.0 it doesn't. So I think this was fixed, perhaps by #3476.

**Comment by hadess** [comment]
> do you have a collection ID configured on any remote?

I don't, and never have, so that was unlikely to be the original problem, 2 years after the fact ;)

```sh
$ flatpak remotes --columns=collection
Collection ID
-
-
-
-
-
-
-
-
-
-
-
-
-
```

Glad your problem is fixed though!
